Dockerfile-test

```diff
@@ -0,0 +1,35 @@
+FROM mono:5
```

Comment:
i see on docker hub that mono 6 is available which may also solve the issue with the "apt-get update" commands where there is no more need for the added "exit 0" commands...